frontend/src/utils/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Filter an object by keys
+ * @param obj
+ * @param keys
+ */
```

Comment:
Just to make it clear that we clone the object and we don't only get the keys.

```suggestion
 * @param keys
 * @return a new object with keys matching filter and corresponding values.
 */
```